deepchem/utils/batch_utils.py

```diff
@@ -0,0 +1,90 @@
+"""
+Utility Functions for computing features on batch.
+"""
+import numpy as np
+
+
+def coulomb_matrix_features(X_b: np.ndarray,
```

Comment:
Let's rename this batch_coulomb_matrix_features

Reply:
Done